libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -37,12 +37,16 @@ What's New in Libc++ 19.0.0?
 
 Implemented Papers
 ------------------
+
 - P2637R3 - Member ``visit``
+- P2652R2 - Disallow User Specialization of ``allocator_traits``
 
 
 Improvements and New Features
 -----------------------------
-TODO
+
+- The ``_LIBCPP_ENABLE_CXX23_USER_SPECIALIZATION_OF_ALLOCATOR_TRAITS`` macro has been added to re-enable user specialization
```

Comment:
There is no need to add a flag for this. The feature has been added in C++23, and not many users will have started relying on it.

Reply:
I thought so, removed it. Thank you for reviewing!